responsive-test-utils/build.gradle.kts

```diff
@@ -19,14 +19,18 @@ plugins {
 }
 
 dependencies {
-    implementation(project(":kafka-client"))
+    // TODO: we should make sure to fail the release of kafka-client
+    // if we don't bump this up - or have some other mechanism to
+    // ensure they are released in tandem - otherwise it is possible
+    // that the latest release of kafka-client will be untestable
+    implementation(libs.bundles.scylla)
```

Comment:
we already had this (two lines below) 😛

Reply:
oops, copy pasta errors :/ I'll clean it up when I reapply the patches